contracts/upgradeable/mocks/extensions/typedMultiAsset/RMRKTypedExternalEquippableMockUpgradeable.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.18;
+
+import "../../../RMRK/extension/typedMultiAsset/RMRKTypedMultiAssetUpgradeable.sol";
+import "../../RMRKExternalEquipMockUpgradeable.sol";
+
+error RMRKTokenHasNoAssetsWithType();
+
+contract RMRKTypedExternalEquippableMockUpgradeable is
+    RMRKExternalEquipMockUpgradeable,
+    RMRKTypedMultiAssetUpgradeable
+{
+    function initialize(address nestableAddress) public override initializer {
+        super.initialize(nestableAddress);
```

Comment:
```suggestion
        RMRKExternalEquipMockUpgradeable.initialize(nestableAddress);
```

I'm actually thinking on not adding upgradeable versions for external equip stuff.. even stop maintaining it. We are not as careful testing that variant :disappointed: